organizational-structure-overview.md

```diff
@@ -64,23 +64,32 @@ flowchart TB
 
     subgraph subWG[Working Groups]
         BP[Best Practices]
+        DEI[Diversity, Equity & Inclusion]
         EU[End Users]
-        IST[Identifying Security Threats]
+        ML[AI/ML Security] 
+        MM[Metrics & Metadata]
         REP[Securing Software Repos]
         SCI[Supply Chain Integrity]
         SCP[Securing Critical Projects]
         ST[Security Tooling]
         VD[Vulnerability Disclosures]
 
         BP ---> Scorecard[Scorecard]
-        IST ---> SecurityInsights[Security Insights]
-        IST ---> SecurityMetrics[Security Metrics]
+        BP ---> BestPracticesBadge[Best Practices Badge]
+        MM ---> SecurityInsights[Security Insights]
+        MM ---> SecurityMetrics[Security Metrics]
+        SCI ---> Gittuf[gittuf]
```

Comment:
```suggestion
        REP ---> Rstuf[Repository Service for TUF]
        SCI ---> Gittuf[gittuf]
```

Reply:
Adding RSTUF as part of the Securing Repos WG

Reply:
missed this one. Thank you!